spec/classes/init_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'spec_helper'
 
 describe 'influxdb' do
-  let(:facts) { { os: { family: 'RedHat' }, identity: { user: 'root' } } }
+  let(:facts) { { identity: { user: 'root' } } }
```

Comment:
ah the following context blocks need to go in `on_supported_os.each do |os, os_facts|`

Reply:
Yep I realized that and am working on fixing it. The tests then fail due to some hardcoded values.